recipe/meta.yaml

```diff
@@ -43,7 +43,7 @@ outputs:
       - gcc                                                          # [osx]
     run_exports:
       strong:
-        - libgfortran >=3.0  # [unix]
+        - libgfortran >=3.0,<4.0.1  # [unix]
```

Comment:
Why 4.0.1 instead of 4.0.0

Reply:
This is what you listed in the PR on staged-recipes:

> Send a PR to https://github.com/conda-forge/conda-forge-repodata-patches-feedstock to fix the libgfortran dependency of existing packages to >=3.0.0,<4.0.1 (done, see conda-forge/conda-forge-repodata-patches-feedstock#7)

Further, per the discussion here (https://github.com/conda-forge/libgfortran-feedstock/pull/16#issuecomment-449405113), for gfortran 7, the library is backwards compatible according to @msarahan. 

If we change this to 4.0.0, then we won't be able to have things built with the later gfortran in the env at the same time with core packages like numpy. This would force us to rebuild the whole stack yet again.

Reply:
> If we change this to 4.0.0, then we won't be able to have things built with the later gfortran in the env at the same time with core packages like numpy. This would force us to rebuild the whole stack yet again.

You'd have to rebuild the whole stack. gfortran 7.3.0 ships `libgfortrans.so.4` while gfortran 4.* ships `libgfortran.so.3`. They have different sonames so need to rebuild all packages.

Reply:
@msarahan addressed this above. We don't need to rebuild the whole stack because the SO's are backwards compatible.

Reply:
No, they are not. They have different SONAMEs meaning the upstream wants you to know that they are not ABI compatible.

Reply:
Done.